src/v3/features/ministry/add_service_time_modal_window/index.tsx

```diff
@@ -0,0 +1,470 @@
+  useEffect(() => {
+    if (convertDurationInSecondsToString(localDurationInSeconds) === 'NaN:NaN') {
+      setLocalDurationInSeconds(0);
+    }
+
+    if (convertDurationInSecondsToString(localCreditHoursDurationInSeconds) === 'NaN:NaN') {
+      setLocalCreditHoursDurationInSeconds(0);
+    }
+  }, [localDurationInSeconds, localCreditHoursDurationInSeconds]);
+
+  useEffect(() => {
+    let studiesCounter = 0;
+    props.bibleStudiesList.forEach((value, index) => {
+      if (checkedLocalStudiesStatesList[index]) {
+        studiesCounter++;
+      }
+    });
+
+    setCountOfStudies(studiesCounter);
+  }, [checkedLocalStudiesStatesList, props.bibleStudiesList]);
+
+  const styleForStyledBox = (theme: Theme) => ({
+    [theme.breakpoints.down('tablet')]: {
+      flexDirection: 'column',
+      '& .MuiBox-root': {
+        width: '100%',
+        maxWidth: 'none',
+        minWidth: 'none',
+      },
+    },
+  });
+
+  return (
+    <StyledModalWindowContainer
+      ref={props.reference}
+      className="pop-up-shadow"
+      sx={{
+        marginLeft: { mobile: '16px', tablet: '24px', desktop: '32px' },
+        marginRight: { mobile: '16px', tablet: '24px', desktop: '32px' },
+        marginTop: '16px',
+        marginBottom: '16px',
+      }}
+    >
+      <Box>
+        <CustomTypography className="h2">{t('tr_addServiceTime')}</CustomTypography>
+        <Box
+          sx={{
+            marginTop: '8px',
+            '.MuiButtonBase-root': { padding: 0 },
+            '.MuiButtonBase-root:hover': { backgroundColor: 'transparent' },
+          }}
+        >
+          <CustomDatePicker view={'button'} onChange={async (value) => setLocalDate(value)} value={new Date()} />
+        </Box>
+      </Box>
+      <StyledRowContainer
+        sx={{ justifyContent: 'space-between', flexDirection: 'column', alignItems: 'stretch' }}
+        ref={styledRowContainerWithCreditHours}
+      >
+        <StyledBox sx={styleForStyledBox}>
+          <CustomTypography className="body-regular">{t('tr_hours')}</CustomTypography>
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              minWidth: '168px',
+              maxWidth: '168px',
+              alignItems: 'center',
+            }}
+          >
+            <MinusButton onClick={decrimentDuration} />
+            <TextField
+              id="standard-basic"
+              variant="standard"
+              sx={{
+                '.MuiInputBase-root::after, .MuiInputBase-root::before': { content: 'none' },
+                '.MuiInputBase-root': {
+                  color:
+                    convertDurationInSecondsToString(localDurationInSeconds) != '00:00'
+                      ? 'var(--black)'
+                      : 'var(--grey-300)',
+                },
+                '.MuiInput-input': {
+                  textAlign: 'center',
+                  fontWeight: '550',
+                  fontFamily: 'Inter',
```

Comment:
@FussuChalice: please can we remove this `fontFamily: 'Inter'` assignment as font is changed dynamically for some languages.